libs/src/services/ethContracts/serviceProviderFactoryClient.js

```diff
@@ -13,37 +13,43 @@ class ServiceProviderFactoryClient extends ContractClient {
     contractRegistryKey,
     getRegistryAddress,
     audiusTokenClient,
-    stakingProxyClient
+    stakingProxyClient,
+    isDebug = false,
   ) {
     super(ethWeb3Manager, contractABI, contractRegistryKey, getRegistryAddress)
     this.audiusTokenClient = audiusTokenClient
     this.stakingProxyClient = stakingProxyClient
+    this.isDebug = isDebug
   }
 
   async registerWithDelegate (serviceType, endpoint, amount, delegateOwnerWallet) {
-    if (!Utils.isFQDN(endpoint)) {
+    if (!this.isDebug && !Utils.isFQDN(endpoint)) {
       throw new Error('Not a fully qualified domain name!')
     }
+
     if (!Number.isInteger(amount) && !Utils.isBN(amount)) {
       throw new Error('Invalid amount')
     }
 
-    let requestUrl = urlJoin(endpoint, 'health_check')
-    let axiosRequestObj = {
-      url: requestUrl,
-      method: 'get',
-      timeout: 1000
-    }
-    const resp = await axios(axiosRequestObj)
-    let endpointServiceType
-    try {
-      endpointServiceType = resp.data.data.service
-    } catch (e) {
-      endpointServiceType = resp.data.service
-    }
-
-    if (serviceType !== endpointServiceType) {
-      throw new Error('Attempting to register endpoint with mismatched service type')
+    /// When in debug mode, skip health check
+    if (!this.isDebug) {
```

Comment:
if we were to do a health check here would it work?

Reply:
it didn't work consistently. could possibly get it to work by introducing some wait times, but would like to avoid all of the retries we had to do during the setup-scripts

Reply:
Will remove this + try adding a wait to mad-dog